samples/minimal.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample configuration provides the minimum configuration required by the DPT scripts.
+# Audit resources will be created locally in the project.
+
+overall:
+  organization_id: '12345678'           # Replace this with your Organization ID, or remove to setup without an organization.
+  billing_account: 000000-000000-000000 # Replace this with your Billing Account.
+  domain: domain.com                    # Replace this with your domain.
+
+projects:
+- project_id: my-project
+  owners_group: my-project-owners@my-domain.com      # Replace this with the owners group for this project.
+  auditors_group: some-auditors-group@my-domain.com  # Replace this with your auditors group.
+  audit_logs:
+    logs_bq_dataset:
+      properties:
+        location: US
+        name: audit_logs
+  resources:
+    gke_clusters:
+    - properties:
+        clusterLocationType: Regional
+        region: us-central1
+        cluster:
+          name: clusterName
+          description: clusterDescription
+          network: "projects/my-project/global/networks/default"
+          subnetwork: "projects/my-project/regions/us-central1/subnetworks/default"
+          nodeConfig:
+            oauthScopes: 
```

Comment:
Just wondering, where do these oauthscopes come from?

Reply:
it's actually default configuration from gke console, seems like it work without them